WARNING:root:c:\Users\kaanh\GitRepos\instahelper-flaskapp\insta.db
WARNING:root:c:\Users\kaanh\GitRepos\instahelper-flaskapp\instahelper_app\insta.db
WARNING:root:c:\Users\kaanh\GitRepos\instahelper-flaskapp\instahelper_app\insta.db
WARNING:root:script
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/4
ERROR:root:Could not like these: ['CFHDsJoHcZ1', 'CFHDQIbgBjr', 'CFHDHfDF81A', 'CFHAzotlt84']
ERROR:root:Liked 0/4
ERROR:root:Could not like these: ['CFHDsJoHcZ1', 'CFHDQIbgBjr', 'CFHDHfDF81A', 'CFHAzotlt84']
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/4
ERROR:root:Could not like these: ['CFHDsJoHcZ1', 'CFHDQIbgBjr', 'CFHDHfDF81A', 'CFHAzotlt84']
ERROR:root:Liked 0/4
ERROR:root:Could not like these: ['CFHDsJoHcZ1', 'CFHDQIbgBjr', 'CFHDHfDF81A', 'CFHAzotlt84']
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/4
ERROR:root:Could not like these: ['CFHDsJoHcZ1', 'CFHDQIbgBjr', 'CFHDHfDF81A', 'CFHAzotlt84']
ERROR:root:Liked 0/4
ERROR:root:Could not like these: ['CFHDsJoHcZ1', 'CFHDQIbgBjr', 'CFHDHfDF81A', 'CFHAzotlt84']
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHMa5vjCqT', 'CFHIzw3Dmel']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHMa5vjCqT', 'CFHIzw3Dmel']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHRIw2AnVJ', 'CFHOA1CAxSi']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHRIw2AnVJ', 'CFHOA1CAxSi']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHRIw2AnVJ', 'CFHOA1CAxSi']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHRIw2AnVJ', 'CFHOA1CAxSi']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHRIw2AnVJ', 'CFHOA1CAxSi']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHRIw2AnVJ', 'CFHOA1CAxSi']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHS4FVDbg8', 'CFHRIw2AnVJ']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHS4FVDbg8', 'CFHRIw2AnVJ']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHTv2ADpbI', 'CFHS4FVDbg8']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHTv2ADpbI', 'CFHS4FVDbg8']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not load https://www.instagram.com/explore/tags/anahtar/?__a=1. Trying again...
WARNING:root:Can not load https://www.instagram.com/explore/tags/anahtar/?__a=1. Contact with support please.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHUCIGhd0p', 'CFHTv2ADpbI']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHUCIGhd0p', 'CFHTv2ADpbI']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHUCIGhd0p', 'CFHTv2ADpbI']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHUCIGhd0p', 'CFHTv2ADpbI']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHVIdlpsyL', 'CFHUCIGhd0p']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHVIdlpsyL', 'CFHUCIGhd0p']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHVIdlpsyL', 'CFHUCIGhd0p']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHVIdlpsyL', 'CFHUCIGhd0p']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHVIdlpsyL', 'CFHUCIGhd0p']
ERROR:root:Liked 0/2
ERROR:root:Could not like these: ['CFHVIdlpsyL', 'CFHUCIGhd0p']
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Problem occured while liking/commenting post.
ERROR:root:Problem occured while liking/commenting post.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not load https://www.instagram.com/explore/tags/anahtar/?__a=1. Trying again...
WARNING:root:Can not load https://www.instagram.com/explore/tags/anahtar/?__a=1. Contact with support please.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not like CFHVIdlpsyL
ERROR:root:Could not post comment for CFHVIdlpsyL
ERROR:root:Problem occured while liking/commenting post.
ERROR:root:Could not like CFHUCIGhd0p
ERROR:root:Could not post comment for CFHUCIGhd0p
ERROR:root:Problem occured while liking/commenting post.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not like CFHVIdlpsyL
ERROR:root:Could not post comment for CFHVIdlpsyL
ERROR:root:Problem occured while liking/commenting post.
ERROR:root:Could not like CFHUCIGhd0p
ERROR:root:Could not post comment for CFHUCIGhd0p
ERROR:root:Problem occured while liking/commenting post.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not like CFHbDRulmCZ
ERROR:root:Could not post comment for CFHbDRulmCZ
ERROR:root:Problem occured while liking/commenting post.
ERROR:root:Could not like CFHa2Wolhmj
ERROR:root:Could not post comment for CFHa2Wolhmj
ERROR:root:Problem occured while liking/commenting post.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not like CFHdg-npUZS
ERROR:root:Could not post comment for CFHdg-npUZS
ERROR:root:Could not like CFHbDRulmCZ
ERROR:root:Could not post comment for CFHbDRulmCZ
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Problem occured while liking/commenting post.
ERROR:root:Problem occured while liking/commenting post.
ERROR:root:Could not load https://www.instagram.com/. Trying again...
ERROR:root:Could not load https://www.instagram.com/. Trying again...
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Problem occured while liking/commenting post.
ERROR:root:Problem occured while liking/commenting post.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C2E340C1C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/199075c6abad9c4959fab0042dea5856/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C2E3454520>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/199075c6abad9c4959fab0042dea5856/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C2E34611C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/199075c6abad9c4959fab0042dea5856/url
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002767E1DC1C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/76aa098481472fd223c232d1bc79e79d/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002767E223520>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/76aa098481472fd223c232d1bc79e79d/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002767E2311C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/76aa098481472fd223c232d1bc79e79d/url
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000230BEA1C1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/285f4d75388b55234b8e8db7c7586673/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000230BEA63520>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/285f4d75388b55234b8e8db7c7586673/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000230BEA711C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/285f4d75388b55234b8e8db7c7586673/url
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000224065BD1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/79703c5a46fdb70de1251d91d4433ac7/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022406604550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/79703c5a46fdb70de1251d91d4433ac7/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000224066131F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/79703c5a46fdb70de1251d91d4433ac7/url
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002C6CFFAC1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/6301e7c589dc2e1f643cc7bd83b5fb7c/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002C6CFFF4550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/6301e7c589dc2e1f643cc7bd83b5fb7c/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002C6D00031F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/6301e7c589dc2e1f643cc7bd83b5fb7c/url
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000153885FD1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/685df28e2d97502cc009bfe8990af447/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000015388644550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/685df28e2d97502cc009bfe8990af447/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000153886531F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/685df28e2d97502cc009bfe8990af447/url
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D0A23AC1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/2462776c7f31c772311c9a27d1c99dbc/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D0A23F4130>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/2462776c7f31c772311c9a27d1c99dbc/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D0A2401370>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/2462776c7f31c772311c9a27d1c99dbc/url
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002096DF1C1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/0eb64303aec9ae5ad1928657b862e25c/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002096DF63100>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/0eb64303aec9ae5ad1928657b862e25c/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002096DF70340>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/0eb64303aec9ae5ad1928657b862e25c/url
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001FB9BF6C1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/a94e7f9b535d5148ca964d7c0dc158fb/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001FB9BFB4130>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/a94e7f9b535d5148ca964d7c0dc158fb/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001FB9BFC1370>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/a94e7f9b535d5148ca964d7c0dc158fb/url
WARNING:root:User bilginhale007 is not logged in.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000027B5B8FD1C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/963bcf98069f595d84bb1a1f7c7d888f/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000027B5B942100>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/963bcf98069f595d84bb1a1f7c7d888f/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000027B5B94F340>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/963bcf98069f595d84bb1a1f7c7d888f/url
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not load https://www.instagram.com. Trying again...
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Username input couldn't loaded
ERROR:root:Password input couldn't loaded
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not load https://www.instagram.com. Trying again...
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:
ERROR:root:
WARNING:root:User bilginhale007 is not logged in.
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:root:Can not load https://www.instagram.com. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C142DDB3D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/75fac1955e83e5a2bfed31e1cb447120/execute/sync
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C143050E80>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/75fac1955e83e5a2bfed31e1cb447120/execute/sync
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001C143050B50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/75fac1955e83e5a2bfed31e1cb447120/execute/sync
ERROR:root:Could not load https://www.instagram.com/explore/tags/ada/?__a=1. Trying again...
WARNING:root:Can not load https://www.instagram.com/explore/tags/ada/?__a=1. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D615B9D460>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/execute/sync
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D616205880>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/execute/sync
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D616205BE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/execute/sync
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D616234040>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D6162342E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D616234430>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/url
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D6162347C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D616234940>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001D616215C10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/62d915f71de8a8790adce3d3c0a34465/url
ERROR:root:Could not load https://www.instagram.com/explore/tags/ada/?__a=1. Trying again...
WARNING:root:Can not load https://www.instagram.com/explore/tags/ada/?__a=1. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022655AB03A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/element
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656114B20>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/element
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656124130>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/element
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656143220>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000226561433D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656143520>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/url
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000226561438B0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656143A30>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656124160>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/1bca2f87e0138c311efba978e03a473b/url
ERROR:root:Could not load https://www.instagram.com/explore/tags/ada/?__a=1. Trying again...
WARNING:root:Can not load https://www.instagram.com/explore/tags/ada/?__a=1. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656143F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/element
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000226561644F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/element
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x00000226561642E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/element
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656164280>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656164940>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656164AF0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/url
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000022656164E80>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002265617A040>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000002265617A1F0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b463fea43d852968ac6208e865cb3db8/url
ERROR:root:Could not load https://www.instagram.com. Trying again...
ERROR:root:Could not load https://www.instagram.com. Trying again...
ERROR:root:Could not load https://www.instagram.com. Trying again...
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:root:Can not load https://www.instagram.com. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020D42411760>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/81733e846b9c0c2c65157099807762b0/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020D42411D00>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/81733e846b9c0c2c65157099807762b0/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020D42411A90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/81733e846b9c0c2c65157099807762b0/url
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:root:Can not load https://www.instagram.com. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020D423E3880>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/073c4f3350f41cac91bd0d6443b1c51f/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020D42411AC0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/073c4f3350f41cac91bd0d6443b1c51f/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000020D424119A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/073c4f3350f41cac91bd0d6443b1c51f/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024CD78D13D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/07ef731209eea774c5d2f5093a19a021/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024CD7F32940>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/07ef731209eea774c5d2f5093a19a021/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024CD7F40BE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/07ef731209eea774c5d2f5093a19a021/url
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024CD7F611C0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/07ef731209eea774c5d2f5093a19a021/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024CD7F61310>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/07ef731209eea774c5d2f5093a19a021/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000024CD7F61460>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/07ef731209eea774c5d2f5093a19a021/url
ERROR:root:Could not load https://www.instagram.com/kaanxhakan. Trying again...
WARNING:root:Can not load https://www.instagram.com/kaanxhakan. Contact with support please.
ERROR:root:Could not load https://www.instagram.com/kaanxhakan. Trying again...
WARNING:root:Can not load https://www.instagram.com/kaanxhakan. Contact with support please.
ERROR:root:Could not load https://www.instagram.com/kaanxhakan. Trying again...
WARNING:root:Can not load https://www.instagram.com/kaanxhakan. Contact with support please.
ERROR:root:Could not load https://www.instagram.com/kaanxhakan. Trying again...
WARNING:root:Can not load https://www.instagram.com/kaanxhakan. Contact with support please.
ERROR:root:Could not load https://www.instagram.com/kaanf. Trying again...
WARNING:root:Can not load https://www.instagram.com/kaanf. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000010DDAE11550>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b2ed0f4cb83a55a033ee8404cc2f034a/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000010DDAE11F10>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b2ed0f4cb83a55a033ee8404cc2f034a/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000010DDAE11C70>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b2ed0f4cb83a55a033ee8404cc2f034a/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000010DDAFB4370>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b2ed0f4cb83a55a033ee8404cc2f034a/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000010DDAE119A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b2ed0f4cb83a55a033ee8404cc2f034a/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x0000010DDAFB4460>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/b2ed0f4cb83a55a033ee8404cc2f034a/url
ERROR:root:Could not load https://www.instagram.com/kaanf. Trying again...
WARNING:root:Can not load https://www.instagram.com/kaanf. Contact with support please.
ERROR:root:Could not load https://www.instagram.com/p/CDgKxMghnSI/. Trying again...
WARNING:root:Can not load https://www.instagram.com/p/CDgKxMghnSI/. Contact with support please.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD128813A0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/execute/sync
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12EF5AC0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/execute/sync
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F04E50>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/execute/sync
ERROR:root:Problem occured while liking/commenting post.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F18190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F18490>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F185E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
ERROR:root:Could not load https://www.instagram.com/p/CAlE0pqB1P_/. Trying again...
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F18970>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F04D90>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F040D0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
ERROR:root:Problem occured while liking/commenting post.
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F188E0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F18AC0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F18C40>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
ERROR:root:Could not load https://www.instagram.com. Trying again...
WARNING:urllib3.connectionpool:Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F18FD0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F43190>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
WARNING:urllib3.connectionpool:Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001DD12F43340>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')': /session/cb2fe9993be9b2e3bca60c71d98a66ea/url
ERROR:root:Could not load https://www.instagram.com/kaanf. Trying again...
WARNING:root:Can not load https://www.instagram.com/kaanf. Contact with support please.
